src/data/equiv/basic.lean

```diff
@@ -751,6 +751,55 @@ def sum_compl {α : Type*} (p : α → Prop) [decidable_pred p] :
   (a : α) (h : ¬ p a) :
   (sum_compl p).symm a = sum.inr ⟨a, h⟩ := dif_neg h
 
+open equiv
+
+variables {ε : Type*} {p : ε → Prop} [decidable_pred p]
+variables (ep ep' : perm {a // p a}) (en en' : perm {a // ¬ p a})
+
+/-- Combining permutations on `ε` that permute only inside or outside the subtype
+split induced by `p : ε → Prop` constructs a permutation on `ε`. -/
+def perm.subtype_congr : equiv.perm ε :=
```

Comment:
Looking at this again - since the definition is so short, should it just be marked reducible, removing the need for lemmas about it? For instance, right now it has to be manually unfolded to compute its `.sign`

Reply:
I don't have a strong opinion about this, but it looks reasonable to me.